src/ripple/app/tx/impl/PayChan.cpp

```diff
@@ -37,86 +37,88 @@ namespace ripple {
 /*
     PaymentChannel
 
-        Payment channels permit off-ledger checkpoints of XRP payments flowing
-        in a single direction. A channel sequesters the owner's XRP in its own
-        ledger entry. The owner can authorize the recipient to claim up to a
-        given balance by giving the receiver a signed message (off-ledger). The
-        recipient can use this signed message to claim any unpaid balance while
-        the channel remains open. The owner can top off the line as needed. If
-        the channel has not paid out all its funds, the owner must wait out a
-        delay to close the channel to give the recipient a chance to supply any
-        claims. The recipient can close the channel at any time. Any transaction
-        that touches the channel after the expiration time will close the
-        channel. The total amount paid increases monotonically as newer claims
-        are issued. When the channel is closed any remaining balance is returned
-        to the owner. Channels are intended to permit intermittent off-ledger
-        settlement of ILP trust lines as balances get substantial. For
-        bidirectional channels, a payment channel can be used in each direction.
+        Payment channels permit off-ledger XRP payments flowing between two
+        accounts in both directions. A channel sequesters the participants' XRP
+        in its own ledger entry. The participants can authorize the counterparty
+        to claim up to a given amount by giving the receiver a signed message
```

Comment:
Unlike unidirectional channels, it looks like a claim is always for the full amount. So we should remove "up to".